server/kubeconfig

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
does this need to be in git?

Reply:
It is a private repo for testing. I will have it removed once we go public

Reply:
Ah it is public :D

Reply:
removed ti